### PR TITLE
Resolves response schemas of actions/functions that return collection

### PR DESCRIPTION
**Pull request by irvinesunday**
Closes https://github.com/microsoft/OpenAPI.NET.OData/issues/116

This PR:
- Updates the `EdmOperationOperationHandler.cs` class by updating the response schema of actions/functions that return a collection.

### CODE REVIEW

File: src/Microsoft.OpenApi.OData.Reader/Operation/EdmOperationOperationHandler.cs [hunk 1]
```diff
@@ -172,6 +172,28 @@ protected override void SetResponses(OpenApiOperation operation)
...
+                        Title = $"Collection of {entityType.Name}",
+                        Type = "object",
+                        Properties = new Dictionary<string, OpenApiSchema>
+                        {
+                            {
+                                "value", Context.CreateEdmTypeSchema(EdmOperation.ReturnType)
+                            }
+                        }
+                    };
+                }
```
Comment (xuzhg) [inline src/Microsoft.OpenApi.OData.Reader/Operation/EdmOperationOperationHandler.cs:191]: It looks you can move these codes into "CreateEdmTypeSchema(operation.ReturnType)".

I am ok now to move it on now.

**Review by xuzhg — APPROVED** [review]
:shipit:

File: src/Microsoft.OpenApi.OData.Reader/Operation/EdmOperationOperationHandler.cs (hunk 1 above)
Comment (irvinesunday) [reply src/Microsoft.OpenApi.OData.Reader/Operation/EdmOperationOperationHandler.cs:191]: I wasn't sure whether the same applies to collection-valued properties. See: https://docs.oasis-open.org/odata/odata-openapi/v1.0/cn01/odata-openapi-v1.0-cn01.html#sec_CollectionValuedProperties

**Comment by baywet** [comment]
@irvinesunday when do you plan to release this fix?

**Comment by irvinesunday** [comment]
> @irvinesunday when do you plan to release this fix?

@baywet just waiting on some validation tests from the PowerShell SDK, then I'll be good to release. Hopefully before the end of the week.
cc: @peombwa

**Comment by baywet** [comment]
Thanks!
Do you have a preview feed/full v1 description so I could look at the result for Kiota and give you some feedback?

Can I suggest you setup milestones on the repo to track the release date and content of any given release?

**Comment by irvinesunday** [comment]
> Thanks!
> Do you have a preview feed/full v1 description so I could look at the result for Kiota and give you some feedback?
> 
> Can I suggest you setup milestones on the repo to track the release date and content of any given release?

You can use the nightly build to test with the latest update in master: https://github.com/microsoft/OpenAPI.NET.OData#nightly-builds

At the moment, most of the fixes done here are usually erratic, I'm not sure whether developing with milestones might be feasible, however it is ideal and definitely worth looking into. Thanks @baywet

**Pull request merged**